test/realtime/server_test.exs

```diff
@@ -568,17 +605,21 @@ defmodule Realtime.ServerTest do
     end
 
     test "returns empty data when the route is not found", %{ets: ets} do
-      assert Server.lookup({ets, {:route_id, "2"}}) == []
+      assert Server.lookup({ets, {:route_id, "3"}}) == []
```

Comment:
suggestion(non-blocking): can we use -1 or something here?
```suggestion
      assert Server.lookup({ets, {:route_id, "-1"}}) == []
```

Reply:
Do you just mean that the `"-1"` would better denote "non-existent route ID" to someone reading the code?

Reply:
Yes, and I think the change here is diff and review overhead that can be avoided. The only reason this was changed was because something else took 2's spot, so ideally this wouldn't need to be changed unless something pertaining to the specific test and the implementation changed.

Reply:
Actually maybe it would be better to be something like this?
```suggestion
      assert Server.lookup({ets, {:route_id, "non-existent-1 "}}) == []
```